### PR TITLE
Modified DM utc to fix build errors

### PR DESCRIPTION
**Pull request by dr-venkman**
@Taejun-Kwon Please review the changes to utc
@sunghan-chang Please check coding format if ok

### CODE REVIEW

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c [hunk 1]
```diff
@@ -40,6 +51,52 @@ extern int working_tc;
 
 static int isConnected = 0;
 
+static slsi_security_config_t *get_security_config(char *sec_type, char *psk)
```
Comment (JunhwanPark) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: Hi, @dr-venkman 
I have a question. The 'security_config' function already exists. Why did you create a new one?
https://github.com/Samsung/TizenRT/blob/master/apps/netutils/wifi/slsi_wifi_utils.c#L34
`getSecurityConfig`

Comment (dr-venkman) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: @JunhwanPark I did not realize it was renamed the way it is now. Will fix as shown above. Thanks.

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c [hunk 2]
```diff
@@ -114,13 +115,13 @@ int wifiAutoConnectInit()
...
 {
 	isConnected = 0;
 	dm_conn_unregister_linkup_cb(linkUpEvent);
 }
 
-static void wifiAutoConnect()
+static int wifiAutoConnect(void)
 {
 	int ret;
 	if ((ret = wifiAutoConnectInit()) == 1) {
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: ret = wifiAutoConnectInit();
if (ret == 1)

Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: Also we don't have to use dhcp_ret , we can use just 'ret'

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c [hunk 3]
```diff
@@ -145,6 +146,12 @@ static void wifiAutoConnect()
 			printf("[AutoConnect]WiFi is not working. Test Canceled\n");
 			return 0;
 		}
+		else {
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: useless 2 else states are here.

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c (hunk 2 above)
Comment (dr-venkman) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: Maybe we don't need ret in the first call itself. 
if (wifiAutoConnectInit() == 1) ...

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c (hunk 3 above)
Comment (dr-venkman) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: Not quite, although I see I miss a return 1 at the end. In that case it is required.

**Comment by dr-venkman** [comment]
Addressed the else states, the first one should return 1, and not -1.

**Comment by sunghan-chang** [comment]
@dr-venkman There are many violations of coding style. Please fix them. And I can not check whole changes about coding style. You must check it by yourself.

File: apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c [hunk 4]
```diff
@@ -144,8 +145,12 @@ static void wifiAutoConnect()
 		if (waitTime <= 0) {
 			printf("[AutoConnect]WiFi is not working. Test Canceled\n");
 			return 0;
+		} else {
```
Comment (Taejun-Kwon) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: you can use just return 1 here

Comment (dr-venkman) [inline apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c]: What if isConnected == 0, then we have to return 0.

**Pull request merged**